<launch>
    <arg name="gps_port" default=""/>
    <arg name="imu_port" default=""/>
    <node name="convert_to_quaternion" pkg="lab4_driver" type="convert_to_quaternion.py">
    </node>
    <node name="gps_driver" pkg="lab4_driver" type="gps_driver.py">
        <param name="gps_port" type="string" value="$(arg gps_port)"/>
    </node>
    node name="imu_driver" pkg="lab4_driver" type="imu_driver.py">
        <param name="imu_port" type="string" value="$(arg imu_port)"/>
    </node>
</launch>
